### ISSUE TITLE
fork CTAssetsController

### ISSUE THREAD

**Issue by ggendre** [cobaltians/cobalt#156]
fork CTAssetsController sur un repository à nous and use it on every existing projects

**Comment by Doc1faux** [comment]
@ggendre Do your work on it/have you done it?

**Comment by ggendre** [comment]
no, i've done nothing. I putted it progress because fork is done. but the change is not done in all our projects
